module/editor/src/Persistence/Query/DbalDraftQuery.php

```diff
@@ -79,6 +80,34 @@ public function getActualDraftId(ProductId $productId): ?ProductDraftId
         return null;
     }
 
+    /**
+     * @param AttributeId $attributeId
+     *
+     * @return ProductDraftId[]
+     */
+    public function getNotAppliedWithAttribute(AttributeId $attributeId): array
```

Comment:
what this function should return as the name of the function is not clear for me.